style.css

```diff
@@ -96,3 +105,188 @@ header {
   width: 96vw;
   top: 0;
 }
+
+/* work section */
+
+.work-section {
+  display: grid;
+  grid-template-columns: auto;
+}
+
+.work-section-title {
+  color: var(--heading-color);
+  font-size: var(--heading-font-size);
+  max-width: 60%;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+}
+
+.work-section-title > span {
+  width: 80px;
+  height: 6px;
+  background-color: #43b886;
+  margin: 0 auto;
+  border-radius: 3px 3px;
+}
+
+/* cards */
+
+.card-container {
+  margin: 0 auto;
+  width: 100%;
+}
+
+.img-placeholder {
+  height: 500px;
+  border-radius: 8% 8% 0 0;
+}
+
+/* work section cards */
+.card-work {
+  max-width: 23rem;
+  background-color: #ebf0ee;
+  height: 554px;
+  margin: 0 auto;
+  margin-top: 10%;
+  display: flex;
+  width: 75%;
+  flex-direction: column;
+  border: solid #d0d9d4 1px;
+  border-radius: 2% 2% 0 0;
+}
+
+h2 {
+  color: #172b4d;
+}
+
+.card-work h2 {
+  color: #3a4a42;
+}
+
+button {
+  background-color: var(--btn-normal);
```

Comment:
- [ ] Please use the same font family provided in Figma desing 
![image](https://user-images.githubusercontent.com/74744787/159041956-171b609b-b838-4019-aaa3-c664d2cbcc1e.png)